test/errors_test.rb

```diff
@@ -164,7 +164,7 @@ def test_unterminated_argument_expression
 
   def test_cr_without_lf_in_percent_expression
     assert_errors expression("%\r"), "%\r", [
-      ["Invalid %% token", 0..3],
+      ["Invalid %% token", 0..2],
```

Comment:
kinda fun we had this test already. Technically this is a very good error test for finding that this string delimiter start was wrong.